EarlyBinding/src/EXPRESS/EXPRESSEntity.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+    This file is part of Expresso, a simple early binding generator for EXPRESS.
```

Comment:
This is not true!

Reply:
I agree - we need to correct the licences ...